feature/isis/link-state-database/feature.textproto

```diff
@@ -0,0 +1,156 @@
+id {
+  name: "isis_link-state-database"
```

Comment:
```suggestion
  name: "isis_lsdb"
```

Reply:
It seems to me that we should keep the name of the Feature Profile consistent with the directory structure, when possible... I think there are some tools that make this assumption.

Reply:
concise and readable suggestion, but the OC path spells this out, so I wanted to align with that for consistency